apps/internal/base/internal/storage/storage.go

```diff
@@ -443,6 +424,66 @@ func (m *Manager) writeAppMetaData(AppMetaData AppMetaData) error {
 	return nil
 }
 
+// RemoveAccount removes all the associated ATs, RTs and IDTs from the cache associated with this account.
+func (m *Manager) RemoveAccount(account shared.Account, clientID string) {
```

Comment:
We should consider keeping the signature as returning an error as this is a public method and we might be able to avoid a change in the future in doing so. Even if it only returns nil for now

Reply:
It is a public method on an internal package.  So users can't use it.  Do you see a reason we would need keep the error saying it will have no user impact to change?

Reply:
ah, you are right. Should be fine as is then.